<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mine_window</class>
 <widget class="QWidget" name="mine_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的百宝箱</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>540</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>484</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>作者博客：fitar.top</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>联系方式：admin@fitarmail.top</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>项目地址：https://github.com/FitarProject/Urchin-Framework</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_custom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>580</width>
      <height>381</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_custom">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>578</width>
       <height>379</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>70</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>自定义功能</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_manage">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>管理</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> 我的功能库</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>580</width>
      <height>81</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>578</width>
       <height>79</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>内置功能</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_export">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导出配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_import">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导入配置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
